src/lib/connectors/shared/async-html.ts

```diff
@@ -48,22 +53,43 @@ export class AsyncHTMLDocument implements IAsyncHTMLDocument {
             return [];
         }
 
-        return nodeIds.map((nodeId) => {
-            return new AsyncHTMLElement(this._nodes.get(nodeId), this, this._DOM); // eslint-disable-line no-use-before-define, typescript/no-use-before-define
-        });
+        const nodes: Array<AsyncHTMLElement> = [];
+
+        for (let i = 0; i < nodeIds.length; i++) {
+            const nodeId = nodeIds[i];
+            const node = this._nodes.get(nodeId);
+
+            if (node) {
+                nodes.push(new AsyncHTMLElement(node, this, this._DOM)); // eslint-disable-line no-use-before-define, typescript/no-use-before-define
+            } else {
+                // This node was added in the DOM and we don't have it cached
+                // TODO: uncomment once chrome 62 is released.
+                // See https://github.com/cyrus-and/chrome-remote-interface/issues/255 for more information
+                // node = await this._DOM.describeNode({ nodeId });
```

Comment:
Meanwhile the release chrome 62, I think a workaround can be call to `const { root: dom } = await this._DOM.getDocument({ depth: -1 });` and add to the cache the new elements.

Reply:
@sarvaje and I spoke over the phone. For now we are going to leave as it is so we don't have to regenerate all the content for our Async Nodes. We might even leave it as it is in the future depending if this is a problem or not.